Cargo.toml

```diff
@@ -35,21 +35,36 @@ version = "0.41.0"
 default-features = false
 features = [
     "mdns",
+    "mplex",
```

Comment:
any particular reason why we need to support multiple muxers? I'm fine with either

Reply:
nope, just copy-paste leftover